manim/mobject/vector_field.py

```diff
@@ -720,20 +748,29 @@ def outside_box(p):
             step = max_steps
             if not step:
                 continue
-            line = VMobject()
+            line = Line()
             line.duration = step * dt
             step = max(1, int(len(points) / self.max_anchors_per_line))
             line.set_points_smoothly(points[::step])
             if self.single_color:
                 line.set_stroke(self.color)
             else:
-                # line.set_stroke([color_func(p) for p in line.get_anchors()])
-                # TODO use color_from_background_image
-                line.color_using_background_image(self.background_img)
-            line.set_stroke(width=self.stroke_width, opacity=opacity)
+                if config["renderer"] == "opengl":
+                    # scaled for compatibility with cairo
+                    line.set_stroke(width=self.stroke_width / 4.0)
+                    norms = [
+                        self.get_norm(self.func(point)) for point in line.get_points()
+                    ]
```

Comment:
iirc `get_norm` was dropped in favour of numpy's `np.linalg.norm`, so it should be called directly here.

plus, `get_points` was deprecated.

```suggestion
                    norms = np.array([
                        np.linalg.norm(self.func(point)) for point in line.points
                    ])
```